src/module/data/effect/base.mjs

```diff
@@ -6,12 +6,49 @@ export default class BaseEffectModel extends foundry.abstract.TypeDataModel {
     type: "base"
   });
 
+  /** @override */
+  static LOCALIZATION_PREFIXES = ["DRAW_STEEL.Effect.base"];
+
+  /** @override */
   static defineSchema() {
     const fields = foundry.data.fields;
     const config = ds.CONFIG;
     return {
-      end: new fields.StringField({choices: config.effectEnds, blank: false}),
-      characteristic: new fields.StringField({choices: config.characteristics, blank: false})
+      end: new fields.StringField({choices: Object.keys(config.effectEnds), blank: true, required: true})
+    };
+  }
+
+  /**
+   * An effect is also temporary if it has the `end` property set even though they have indeterminate lengths
+   * @returns {true | null}
+   */
+  get isTemporary() {
+    if (this.end) return true;
+    else return null;
```

Comment:
Any specific reasoning for `null`? Seems to me it can return `false` just fine.

Reply:
The document class is using null coalescing, so `false` would be a rejection of whatever the normal determination is.